src/mobot_client/payments/__init__.py

```diff
@@ -0,0 +1,232 @@
+# Copyright (c) 2021 MobileCoin. All rights reserved.
```

Comment:
I think in general __init__.py should be empty except for importing public objects or running initialization code, buy my understanding may be out of date (looking at https://docs.python.org/3/tutorial/modules.html#packages)

Reply:
Sure, I moved them to inner class files.